torch/distributed/checkpoint/_async_process_executor.py

```diff
@@ -224,22 +233,23 @@ def _checkpointing_subprocess(
                     f"Received async checkpoint request with id={obj.checkpoint_request_id.checkpoint_id}"  # noqa: G004
                 )
 
-                response = _AsyncCheckpointProcess._execute_save(
-                    obj.staged_state_dict,
-                    checkpoint_request_id=obj.checkpoint_request_id,
-                    storage_writer=obj.storage_writer,
-                    planner=obj.planner,
-                )
-                parent_conn.send(response)
-                logger.info(
-                    f"Submitted checkpoint save request for checkpoint_id={obj.checkpoint_request_id}"  # noqa: G004
-                )
-        except BaseException as e:
-            logger.error(
-                f"Checkpoint background process encountered an exception: {e}"  # noqa: G004
-            )
-            parent_conn.send(e)
-            raise
+                try:
+                    response = _AsyncCheckpointProcess._execute_save(
+                        obj.staged_state_dict,
+                        checkpoint_request_id=obj.checkpoint_request_id,
+                        storage_writer=obj.storage_writer,
+                        planner=obj.planner,
+                    )
+                    parent_conn.send(response)
+                    logger.info(
+                        f"Completed checkpoint save request for checkpoint_id={obj.checkpoint_request_id}"  # noqa: G004
```

Comment:
Please avoid unnecessary fstring formatting particulraly for info calls, use lazy % formatting